content/docs/rfcs/71/README.md

```diff
@@ -0,0 +1,436 @@
+
+Data disclosed
+- The `SHAKE-256` of the `chat_id` the notification is to be sent for
+- The cypher text of the message
+- The `SHAKE-256` of the public key of the sender
+- The type of notification
+
+### PushNotificationRequest:
+`requests`: a list of PushNotification `message_id`: the [status message id](https://specs.status.im/spec/6)
+
+Data disclosed
+- The status `message_id` for which the notification is for
+
+### PushNotificationResponse: 
+`message_id`: the `message_id` being notified on. `reports`: a list of `PushNotificationReport`.
+
+### PushNotificationReport:
+`success`: whether the push notification was successful. `error`: the type of the error in case of failure. `public_key`: the public key of the user being notified. `installation_id`: the `installation_id` of the user being notified.
+
+## Anonymous Mode
+In order to preserve privacy, the client MAY provide anonymous mode of operations to propagate information about the user. A client in anonymous mode can register with the server using a key that is different from their chat key. This will hide their real chat key. This public key is effectively a secret and SHOULD only be disclosed to clients approved to notify a user. 
+
+- A client MAY advertise the access token on the [contact-code topic](https://specs.status.im/spec/6) of the key generated. 
+- A client MAY share their public key contact updates in the [protobuf record](https://developers.google.com/protocol-buffers/). 
+- A client receiving a push notification public key SHOULD listen to the contact code topic of the push notification public key for updates.
+
+The method described above effectively does not share the identity of the sender nor the receiver to the server, but MAY result in missing push notifications as the propagation of the secret is left to the client. This can be mitigated by [device syncing](https://specs.status.im/spec/6), but not completely addressed.
+
+# Security/Privacy Considerations
+If no anonymous mode is used, when registering with a push notification service a client disclose:
+- The devices that will receive notifications.
+- The chat key.
+
+A client MAY disclose:
+- The hash of the `chat_id` they want to filter out.
+
+When running in anonymous mode, the client’s chat key is not disclosed. 
+
+When querying a push notification server a client will disclose:
+- That it is interested in sending push notification to another client, but querying client’s chat key is not disclosed.
+When sending a push notification a client disclose:
+- The `shake-256` of the `chat_id`.
+
+# Copyright
+
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+
+# References
+
+
+
+
```

Comment:
May want to add the normative and informative references here - link back to status specs, mention other status specs that have been ported

Reply:
The references used in the RFC link to the Status Specification website. Should i change them to the status specs github repository?